projects/components/src/copy-to-clipboard/copy-to-clipboard.component.ts

```diff
@@ -52,7 +52,7 @@ export class CopyToClipboardComponent implements OnInit, OnDestroy {
   public size?: ButtonSize = ButtonSize.Small;
 
   @Input()
-  public icon?: IconType = IconType.CopyToClipboard;
+  public icon?: IconType = IconType.ContentCopy;
```

Comment:
If this is different than copy to clipboard and meant to replace it, can we remove the old one

Reply:
I have to check with Michael on this, but this one looks better and is coming from material directly.